intellij_idea_set_up.md

```diff
@@ -0,0 +1,22 @@
+# IntelliJ IDEA settings
+
+### Installing the Bazel plugin
+Install it from within the IDE (`Settings > Plugins > Marketplace`, and search for **Bazel**).
+  
+  See another option to install the Bazel plugin - https://ij.bazel.build/docs/bazel-plugin.html#getting-started.
```

Comment:
`See another option to install the Bazel plugin->If that doesn't work, see alternative option`

Reply:
Done.